### PR TITLE
Tracing put tests

### PR DESCRIPTION
**Pull request by jgpruitt**
### Add tests for tracing put/get db functions

Tests:
- put_schema_url
- put_instrumentation_lib
- put_tag_key
- put_tag
- get_tag_map
- put_operation

Also removed get_operation as it was unused.

### CODE REVIEW

**Comment by jgpruitt** [comment]
ready for review!

**Review by bitwalker — APPROVED** [review]
LGTM! Granted, I'm coming into this without much context, but I did some poking around enough to feel confident enough in a review.

File: pkg/version/version.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ var (
 	// It is customary to bump the version by incrementing the numeral after
 	// the `dev` tag. The SQL migration script name must correspond to the /new/ version.
 
-	Promscale                           = "0.7.0-beta.1.dev.3"
+	Promscale                           = "0.7.0-beta.1.dev.4"
```
Comment (VineethReddy02) [inline pkg/version/version.go:41]: Any specific reason why are we changing the dev version on every PR? Doesn't this add friction as other PR's will collide with same `dev.X` tags and requires to rebase as other PR's get in with the same tag?

**Review by VineethReddy02 — APPROVED** [review]

File: pkg/version/version.go (hunk 1 above)
Comment (jgpruitt) [reply pkg/version/version.go:41]: As I understand it, that last number in the `dev.X` controls which SQL scripts get run from the upgrade folder. In other words, if you add a new upgrade script (like 4-drop_get_operation.sql) then you have to increase the version to `dev.4` or your upgrade script won't run.

Comment (jgpruitt) [reply pkg/version/version.go:41]: I could be mistaken, and/or not answering your question exactly.

**Pull request merged**